src/real/transcScript.sml

```diff
@@ -727,6 +727,16 @@ QED
 (* Some properties of roots (easier via logarithms)                          *)
 (*---------------------------------------------------------------------------*)
 
+Definition root:
```

Comment:
Couldn't `root` stay in real?  It's no more transcendental than `sqrt`?

Reply:
That's because so far we don't have other `ROOT` theorems (with elementary proofs) moved into `realTheory`, and a definition itself is hardly useful without its supporting theorems.   In general, I think it's not a very good style to isolate a definition from all its properties.

If, in the future, some or all `ROOT` theorems could be re-proved (not as easy as the case of `sqrt`) without using transcendental functions, then of course they should be moved out of `transcTheory`.   But I'm fine to move `root` back if you insist on...

Reply:
OK; this is fair.